app/views/scenarios/index.html.erb

```diff
@@ -21,7 +21,7 @@
         <% @scenarios.each do |scenario| %>
           <tr>
             <td>
-              <%= scenario_label(scenario, content_tag(:i, '', class: 'glyphicon glyphicon-font')) %>
+              <%= scenario_label(scenario, icon(scenario.icon)) %>
```

Comment:
Right now the drop down box default to the first icon `adjust` so it will always have something

![screen shot 2016-05-17 at 10 22 23 am](https://cloud.githubusercontent.com/assets/1547934/15325748/5e3e8868-1c19-11e6-977f-1835e8127a19.png)

But if I were to remove the dropdown, then without the icon will just show the color of the scenario with no icon

![screen shot 2016-05-17 at 10 28 20 am](https://cloud.githubusercontent.com/assets/1547934/15325966/32bd6b40-1c1a-11e6-8893-7e51b5e83ae6.png)